pages/api/performances/nominate.js

```diff
@@ -36,28 +36,37 @@ export default async (req, res) => {
   // TODO for now we assume frontend will filter correctly and only
   // eligible awards will show up for validation
   try {
+    // First clear all the current nominations
+    await prisma.awardPerformance.deleteMany({
+      where: {
+        performance_id: parseInt(performanceID),
+        user_id: parseInt(judgeID) || currentUserID,
+      },
+    });
+
     const awardPerformances = await prisma.$transaction(
```

Comment:
My PR also made some changes here so it would be good to test again on staging right after both this PR and my PR are merged

Reply:
Good idea.

Reply:
Likely you may want to rebase/pull the changes from staging into here now.

Reply:
Both of our PRs have gone in.